src/store/plugins/loading-bar-management.plugin.js

```diff
@@ -10,7 +10,7 @@ const loadingBarManagementPlugin = (store) => {
         const isLoadingBarVisible = state.ui.showLoadingBar
         // for now, we only check if app is ready and printing status
         // TODO: any pending request to the backend should trigger the loading bar
-        const shouldLoadingBarBeVisible = !state.app.isReady || state.print.printingStatus
+        const shouldLoadingBarBeVisible = !state.app.isReady
```

Comment:
If the printStatus is removed, how will the loading bar be shown?

Reply:
I should add a todo in the usePrint composable to tackle that.
The idea is that the composable itself will dispatch something (the source it is waiting on) and then subsequently dispatch once more when it is done. The loading bar will be shown whenever a source is not done loading, meaning we will be removing this loading bar management plugin altogether.